x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plugin/TextFormat.java

```diff
@@ -235,7 +247,7 @@ String contentType(RestRequest request) {
         }
 
         @Override
-        String maybeEscape(String value) {
+        String maybeEscape(String value, Character __) {
```

Comment:
Why did you add that second argument? I don't see it being used, or am I missing something :) ?

Reply:
The second param of `maybeEscape`, the delimiter, is only used for the `CSV` value, but not for the `TSV`.

Reply:
Of course, sorry, I was just confused.